app/routes.py

```diff
@@ -1,2 +1,291 @@
+
+    return make_response({
+        "id": video.video_id
+    }, 200)
+
+def is_int(value):
+    try:
+        return int(value)
+    except ValueError:
+        return False
+
+@rentals_bp.route("/check-out", methods=["POST"], strict_slashes=False)
+def rent_videos():
+    
+    request_body = request.get_json()
+    if "customer_id" not in request_body and "video_id" not in request_body: 
+            return make_response(jsonify({"details": "Invalid data"}), 400)
+    
+    if not is_int(request_body["customer_id"]):
+        return make_response(jsonify({"details": "Invalid data"}), 400)
+    
+    if not is_int(request_body["video_id"]):
+        return make_response(jsonify({"details": "Invalid data"}), 400)
+
+    video_id = request_body["video_id"] 
+    video = Video.query.get(video_id) 
+    if not video:
+        return make_response(jsonify({"details": "Invalid data"}), 404) 
+
+    if video.available_inventory <= 0:
+        return make_response(jsonify({"details": "Invalid data"}), 400) 
+
+
+    customer_id = request_body["customer_id"] 
+    customer = Customer.query.get(customer_id) 
+
+    if not customer:
+        return make_response(jsonify({"details": "Invalid data"}), 404) 
+
+    rental = Rental.check_out(customer_id, video_id)
+    return jsonify(rental.to_json()), 200
+
+
+@rentals_bp.route("/check-in", methods=["POST"], strict_slashes=False)
+def return_videos():
+    
+    request_body = request.get_json()
+    if "customer_id" not in request_body and "video_id" not in request_body:
+            return make_response(jsonify({"details": "Invalid data"}), 400)
+
+    if not is_int(request_body["customer_id"]):
+        return make_response(jsonify({"details": "Invalid data"}), 400)
+    
+    if not is_int(request_body["video_id"]):
+        return make_response(jsonify({"details": "Invalid data"}), 400)
+
+    video_id = request_body["video_id"]
+    video = Video.query.get(video_id) 
+    if not video:
+        return make_response(jsonify({"details": "Invalid data"}), 404) 
+
+    customer_id = request_body["customer_id"] 
+    customer = Customer.query.get(customer_id) 
+
+    if not customer:
+        return make_response(jsonify({"details": "Invalid data"}), 404) 
+        
+
+    rental = Rental.check_in(customer_id, video_id)
+    print(rental)
+    if rental == None:
+        return make_response(jsonify({"details": "Invalid data"}), 400)
+
+    return jsonify(rental.to_dict()), 200
+
+
+@customers_bp.route("/<customer_id>/rentals", methods=["GET"], strict_slashes=False)
+def custumers_check_outs(customer_id):
+
+    customer = Customer.query.get(customer_id)
+
+    if Customer.query.get(customer_id) is None:
+        return make_response({"details": "Invalid data"}, 404)
+
+    rentals = db.session.query(Rental)\
+        .join(Customer, Customer.customer_id==Rental.customer_id)\
+        .join(Video, Video.video_id==Rental.video_id)\
+        .filter(Customer.customer_id==customer_id)
```

Comment:
This works, but you can also use the relationships you make in the model to do, `customer.rentals` and `video.rentals`.